taboos.json

```diff
@@ -388,5 +388,246 @@
+            },
+            {
+                "code": "07308",
+                "xp": -2
+            },
+            {
+                "code": "08032",
+                "xp": 2
+            },
+            {
+                "code": "08099",
+                "xp": 2
+            },
+            {
+                "code": "07269",
+                "text": "This card’s ability gains: “Remove A Watchful Peace from the game.”"
+            },
+            {
+                "code": "07301",
+                "text": "This card’s ability gains: “Remove Hallow from the game.”"
+            },
+            {
+                "code": "01033",
+                "text": "This card’s [reaction] ability now reads: “After you successfully investigate, exhaust Dr. Milan Christopher...”"
+            },
+            {
+                "code": "02002",
+                "text": "This card’s [reaction] ability gains: “(Limit once per round.)”"
+            },
+            {
+                "code": "02111",
+                "text": "This card gains: “Group limit 2 copies of Delve Too Deep in the victory display.”"
+            },
+            {
+                "code": "02229",
+                "text": "At the end of its ability, this card gains: “(max once per round).”"
+            },
+            {
+                "code": "02266",
+                "text": "This card gains: “Max once per round.”"
+            },
+            {
+                "code": "03029",
+                "text": "This card’s ability now reads: “Put a level 0-3 [[Item]] asset into play from your hand.”"
+            },
+            {
+                "code": "03315",
+                "exceptional": 1,
+                "text": "This card gains the Exceptional keyword."
+            },
+            {
+                "code": "04309",
+                "text": "At the end of its ability, this card gains: “Remove All In from the game.”"
+            },
+            {
+                "code": "05116",
+                "text": "This card’s [action] ability is now a [fast] ability."
+            },
+            {
+                "code": "05188",
+                "text": "This card’s [action] ability is now a [fast] ability."
+            },
+            {
+                "code": "05189",
+                "text": "This card’s [action] ability is now a [fast] ability."
+            },
+            {
+                "code": "06195",
+                "text": "This card’s [action] ability now reads: “[action] Spend 1 ammo: <b>Fight.</b> You get +2 [combat] for this attack. If a chaos token with a non-negative modifier is revealed during this attack, this attack deals +2 damage.”"
+            },
+            {
+                "code": "02026",
+                "deck_limit": 0,
+                "text": "Forbidden."
+            },
+            {
+                "code": "02029",
+                "text": "This card now reads \"a symbol\" instead of the list of indicated symbols."
```

Comment:
NB and now you've moved back to \" instead of curled quotes.

Reply:
```suggestion
                "text": "This card now reads “a symbol” instead of the list of indicated symbols."
```

Reply:
I'll make them all consistent with smart quotes